src/Microsoft.DotNet.Wpf/specs/tooltip.md

```diff
@@ -0,0 +1,305 @@
+
+All the changes described here are required to appear in .NET 6.0.
+
+The behavior changes will also appear in servicing updates for earlier versions as follows:
+- .NET 4.6-4.7.2, .NET 4.8:  the next biannual accessibility update (expected summer 2021)
+- .NET Core 3.1, .NET 5.0:  same time frame as above, patch version TBD
+
+The API changes (and behavior that depends on them) are only in .NET 6.0;
+servicing updates cannot include changes to public API.
+
+In the servicing updates, the new behavior is opt-in by setting
+`Switch.UseLegacyAccessibilityFeatures.5` to false, as described [here](https://docs.microsoft.com/en-us/dotnet/framework/whats-new/whats-new-in-accessibility).
+
+
+
+# Conceptual pages (How To)
+
+## Declaring a tooltip
+You can declare a tooltip in XAML or code as described in [TO].
+Starting in .NET 6.0, your declarations can also use the properties
+`ToolTip.ShowsToolTipOnKeyboardFocus`, and
+`ToolTipService.ShowsToolTipOnKeyboardFocus`.
+
+## Opening a tooltip
+When a parent element declares a tooltip, there are four ways to open (show) the tooltip:
+
+1. Hover the mouse over the parent.
+2. Move keyboard focus to the parent.
+3. Type a keyboard shortcut.
+4. Open the tooltip programmatically.
+
+Unless mentioned otherwise below, the tooltip is shown immediately and its size and position are governed by the placement properties described in [TO].
+
+### Hover
+A hover occurs when the mouse enters the parent element and stays there for a certain time, without any other activity.
+The tooltip is opened after the hover time, whose value is a function of the `InitialShowDelay` and `BetweenShowDelay` properties, and whether another tooltip is already showing [TO].
+
+### Keyboard focus
+When you use keyboard navigation to move focus to the parent element, the tooltip opens.
+(Using the mouse to move focus does not show the tooltip.)
+You can override this behavior using the `ShowsToolTipOnKeyboardFocus` properties.
+
+### Keyboard shortcut
+When the parent has keyboard focus, typing Ctrl+Shift+F10 opens the tooltip.
+
+When the tooltip opens due to either of the keyboard actions, a [PlacementMode](https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.primitives.placementmode?view=net-5.0) value of `Mouse` or `MousePoint` is replaced by `Bottom`.
+
+### Programmatically
+[TODO: Check if this is possible.  If so, describe it]
+
+## Closing a tooltip
+A tooltip closes for one of the following reasons.
+
+- **Timeout**. When the tooltip's `ShowDuration` time expires, the tooltip closes.
+- **Hover ends**.  When the tooltip opened due to hover, moving the mouse outside a "safe region" closes the tooltip.
+The safe region is the union of the parent element, the tooltip, and the mouse buffer zone connecting the two (this is explained further in the next section).
+- **Keyboard Shortcut**.  When the parent has keyboard focus, typing Ctrl+Shift+F10 or simply Ctrl (without any other keys) closes the tooltip.
```

Comment:
If I tab to the parent and it opens, then press Control, does it close?

Editing note: I don't know that the docs.microsoft.com page for APIs generally goes into full detail about the specific user experience. From the app developer's POV the framework interacts with the end user how it sees best, maybe differently across releases, but it's nothing the app needs to write code fore. 

It's great it be written down here though. So I usually put user experience and implementation details in the appendix. But here is fine too.

Reply:
>If I tab to the parent and it opens, then press Control, does it close?

Well yes.  Tab to the parent puts focus on the parent, so pressing Ctrl closes the tooltip.  (Well, pressing and releasing Ctrl without touching any other keys, to be super precise.)

The API reference pages don't have this kind of detail, but the overview pages do.  And the audience for this spec includes the folks who want us to comply with [MAS] and [FD] - shunting this off to an appendix sends them the wrong message (that compliance is an afterthought).  After all, this spec is primarily about user experience.